Src/FluentAssertions.Core.Dotnet/project.json

```diff
@@ -1,16 +1,13 @@
 ﻿{
   "supports": {
-    "net46.app": {},
-    "uwp.10.0.app": {},
-    "dnxcore50.app": {}
   },
   "dependencies": {
-    "Microsoft.NETCore": "5.0.0",
-    "Microsoft.NETCore.Portable.Compatibility": "1.0.0"
+    "NETStandard.Library": "1.6.0",
+    "System.Reflection.TypeExtensions": "4.1.0" 
```

Comment:
you use the reverse package search to find what package a type is in.

http://packagesearch.azurewebsites.net/